src/conversion/configurationdefinedconverters.js

```diff
@@ -0,0 +1,310 @@
+ *
+ *		<p>An example text with some big elements:
+ *			<span class="text-big>one</span>,
+ *			<span data-size="big>two</span>,
+ *			<span class="font font-huge>three</span>,
+ *			<span style="font-size:18px>four</span>
+ *		<p>
+ *
+ * into `fontSize` model attribute with 'big' value set so after rendering it the output HTML will be cleaned up:
+ *
+ *		<p>An example text with some big elements:
+ *			<span class="text-big>one</span>,
+ *			<span class="text-big>two</span>,
+ *			<span class="text-big>three</span>,
+ *			<span class="text-big>four</span>
+ *		<p>
+ *
+ * @param {String} attributeName Attribute name to which convert.
+ * @param {module:engine/conversion/configurationdefinedconverters~ConverterDefinition} definition A conversion configuration.
+ * @param {Array.<module:engine/conversion/viewconversiondispatcher~ViewConversionDispatcher>} dispatchers
+ */
+export function viewToModelAttribute( attributeName, definition, dispatchers ) {
+	const { model: attributeValue, viewDefinitions } = parseConverterDefinition( definition );
+
+	const converter = prepareViewConverter( dispatchers, viewDefinitions );
+
+	converter.toAttribute( () => ( {
+		key: attributeName,
+		value: attributeValue
+	} ) );
+}
+
+// Prepares a {@link module:engine/conversion/configurationdefinedconverters~ConverterDefinition definition object} for building converters.
+//
+// @param {module:engine/conversion/configurationdefinedconverters~ConverterDefinition} definition An object that defines view to model
+// and model to view conversion.
+// @returns {Object}
+function parseConverterDefinition( definition ) {
+	const model = definition.model;
+	const view = definition.view;
+
+	const viewDefinition = typeof view == 'string' ? { name: view } : view;
+
+	const viewDefinitions = definition.acceptsAlso ? definition.acceptsAlso : [];
+
+	viewDefinitions.push( viewDefinition );
+
+	return { model, viewDefinition, viewDefinitions };
+}
+
+// Helper method for preparing a view converter from passed view definitions.
+//
+// @param {Array.<module:engine/conversion/viewconversiondispatcher~ViewConversionDispatcher>} dispatchers
+// @param {Array.<module:engine/view/viewelementdefinition~ViewElementDefinition>} viewDefinitions
+// @returns {module:engine/conversion/buildviewconverter~ViewConverterBuilder}
+function prepareViewConverter( dispatchers, viewDefinitions ) {
+	const converter = buildViewConverter().for( ...dispatchers );
+
+	for ( const viewDefinition of viewDefinitions ) {
+		converter.from( definitionToPattern( viewDefinition ) );
+
+		if ( viewDefinition.priority ) {
+			converter.withPriority( viewDefinition.priority );
+		}
+	}
+
+	return converter;
+}
+
+// Converts viewDefinition to a matcher pattern.
+//
+// @param {module:engine/view/viewelementdefinition~ViewElementDefinition} viewDefinition
+// @returns {module:engine/view/matcher~Pattern}
+function definitionToPattern( viewDefinition ) {
```

Comment:
Instead if translating syntaxes, we should make them the same (change matcher pattern to fit definition pattern).